client/www/index.html

```diff
@@ -253,38 +379,164 @@ <h1>Update Profile</h1>
+                        <td><span>Email</span></td>
+                        <td><span>SMS</span></td>
+                    </tr>
+                    <tr>
+                        <td class="checkbox-holder"><span class="custom-checkbox"><input type="checkbox" class="individual-checkbox"/></span></td>
+                        <td><span>Mahammad Shaik</span></td>
+                        <td><span>Brisbane</span></td>
+                        <td><span>Haircut</span></td>
+                        <td><span>Y</span></td>
+                        <td><span>N</span></td>
+                    </tr>
+                </table>
+                <div class="pagination">
+                    <a href="#" class="active">1</a>
+                    <a href="#">2</a>
+                    <a href="#">3</a>
+                    <!-- TO-DO Need to implement based on records -->
+                </div>
 
-        </div>
-    </div>
+                <footer data-role="footer" data-position="fixed" id="foort">
+                    SEND MESSAGE
+                </footer>
+            </div>
+
+            <!-- List of Volunteers Page-->
+            <div data-role="page" id="volunteer-list-page">
+                <div data-role="header" data-position="fixed">
+                    <h1>Volunteers List</h1>
+                </div>
+                <div class="filter-container">
+                    <form id="filter-volunteers">
+                        <label for="select-locations">Locations</label>
+                        <select data-native-menu="false" multiple="multiple" name="locations" data-mini="true" data-required="false"><!--data-param="prefLocation"-->
+                        <!-- ADDED WITH JS -->
+                    </select>
+
+                        <label for="select-services">Service Categories</label>
+                        <select multiple="multiple" data-native-menu="false" name="services" data-mini="true" data-required="false"><!--data-param="prefServiceCategories"-->
+                        <!--ADDED WITH JS-->
+                    </select>
+                    </form>
+                    <!-- END filter form-->
+                </div>
+                <div class="my-table-container">
+                    <table>
+                        <tr class="headers">
+                            <td class="checkbox-holder"><span><input type="checkbox" class="select-all"/></span></td>
+                            <td><span>Name</span></td>
+                            <td><span>Location</span></td>
+                            <td><span>Services</span></td>
+                            <td><span>Email</span></td>
+                            <td><span>SMS</span></td>
+                        </tr>
+                        <tr>
+                            <td class="checkbox-holder"><span class="custom-checkbox"><input type="checkbox" class="individual-checkbox"/></span></td>
+                            <td><span>Mahammad Shaik</span></td>
```

Comment:
We will want to look at solution for longer names here. It doesn't stack well on the smaller screen
![image](https://github.com/sudohex/VolunteerVenture/assets/130754701/e22746f7-d47f-40dc-aef6-9ca695ff0e67)